README.md

```diff
@@ -1,20 +1,118 @@
 FACT-Finder Web Components
 ==========================
+Predictive Basket branch (experimental)
+---------------------------------------
+
+This branch introduces the `ff-predictive-basket` element for easy integration with the new FACT-Finder Predictive Basket feature. To use it, you need FACT-Finder version **7.3** or higher.
+
+_Note that this feature is at an experimental stage and changes may occur._
+
+### The Predictive Basket Element
+
+The following listing shows the newly introduced element. The element itself does not have a visual component and must be used in combination with an `ff-record-list` in the same manner as `ff-recommendation`. See further below for more details.
+
+```html
+<ff-predictive-basket user-id="123456"
+                      max-results="10"
+                      certainty="60"
+                      blacklist="art-id1,art-id2">
+    <!-- requires ff-record-list here to work -->
+</ff-predictive-basket>
+```
+
+Once the element is correctly initialized, it will automatically issue a request to FACT-Finder and display the result. This will happen only once per page load. If you need to refresh the displayed data, refer to the `getPredictions()` method below.
+
+#### Attributes
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+user-id   | yes      | String | _(none)_ | `userId`
+
+The ID of the user for whom products shall be predicted. If the user ID is not specified, no request to FACT-Finder will be issued.
+
+
+
+Attribute | Required | Type | Default | JS property
+--------- | -------- | ---- | ------- | -----------
+max-results | optional | Number |  _(unlimited)_ | `maxResults`
+
+The maximum amount of products to be returned from FACT-Finder. To receive all predictions, omit this attribute or set its value to `-1`.
```

Comment:
maybe 10 or 5 as default?

Reply:
Default is according to FF API

Reply:
then we should mention that or people end up confused